### ISSUE TITLE
parameter tuning in model training

### ISSUE THREAD

**Issue by woshixyr** [LianpoWang/ICDICNet#17]
When training on different datasets, do hyperparameters such as learning rate and batch size need to be adjusted specifically? Are there any verified recommended settings for reference?
